paimon-hive/paimon-hive-connector-common/src/test/java/org/apache/paimon/hive/HiveCatalogITCaseBase.java

```diff
@@ -1155,7 +1155,7 @@ public void testMarkDone() throws Exception {
                                 + "'partition.idle-time-to-done'='1 s',"
                                 + "'partition.time-interval'='1 d',"
                                 + "'metastore.partitioned-table'='true',"
-                                + "'partition.mark-done-action'='done-partition,success-file'"
+                                + "'partition.mark-done-action'='done-partition,success-file,mark-event'"
```

Comment:
Can you validate this in below testing codes?

Reply:
done